docs/AdvancedNetworkPolicy.md

```diff
@@ -8,19 +8,38 @@
 
 # Advanced Network Policy
 
-Calico endpoints are assigned their network policy by configuring them with a policy profile.  In the previous examples we created profiles and assigned endpoints to them.  By default, `calicoctl profile add` adds default config to profiles so endpoints with the profile can all communicate with one another.  In this section, we look at how to customize policy profiles for more advanced policy.
+Calico endpoints are assigned their network policy by configuring them with a 
+policy profile.  
+
+In the [Calico with no Docker networking getting started guide](getting-started/no-docker-networking/GettingStarted.md)
+we created profiles and assigned endpoints (a container interface) to them.
+In the [Calico as a Docker network plugin](getting-started/docker-network-plugin/GettingStarted.md),
+the profiles are created under-the-covers as part of the Docker network
+creation (avialable in natively Docker 1.9 and above).
+
+By default, a Calico profile is created with rules and tags (identifiers) such
+that endpoints with the same profile can all communicate with one another.  
+In this section, we look at how to customize policy profiles for more advanced 
+policy.
 
 ## Overview
 
 A policy profile comprises two elements: *tags* and *rules*.  
 
-*Tags* identify different groups or sets of Calico endpoints.  A tag might represent a role or network permission, for example, you might define a tag `db_access` that represents permission to send data to your database containers on port 3590.
+*Tags* identify different groups or sets of Calico endpoints.  A tag might
+represent a role or network permission, for example, you might define a tag
+`db_access` that represents permission to send data to your database containers
+on port 3590.
 
-*Rules* are the policy statements that Calico's firewall will enforce for your endpoints.
+*Rules* are the policy statements that Calico's firewall will enforce for your
+endpoints.
 
 ## Worked Example
 
-This example assumes you have a working Calico-enabled cluster, as described in the demonstration guides for Calico with [Docker default networking](docs/getting-started/default-networking/Demonstration.md) or [Docker libnetwork](docs/getting-started/libnetwork/Demonstration.md).
+This example assumes you have a working Calico-enabled cluster, as described in
+the getting started guides for Calico with
+[Calico with no Docker networking getting started guide](getting-started/no-docker-networking/GettingStarted.md)
+or [Calico as a Docker network plugin](getting-started/docker-network-plugin/GettingStarted.md).
```

Comment:
Seems a bit funny to have links to these guides both here and 2 sections up on this same page.  Should they just be either here or above?